tests/Tools/vmOpsTool/InMemoryVMWareImplPlatformTests.java

```diff
@@ -1,12 +0,0 @@
-public class InMemoryVMWareImplPlatformTests extends VMWarePlatformTests {
-
-    @Override
-    public IVMWare getVmWareImpl() {
-        return new InMemoryVMWareImpl();
-    }
-

```

Comment:
Don't remove this. The idea here is In Memory testing where In memory implementation mimics actual platform behaviour. These tests run fast, so we run these tests before any check in and after check in a full run happens on actual platform in our case here it is vCenter. Then we will see if our assumption about platform behavior is same or not, if it turned out to be wrong we will update in memory implementation. So if there is platform behaviour change our in memory tests should catch actual product issues. That is the idea here.